server/common/utils/userUtils.js

```diff
@@ -29,7 +34,40 @@ const extractUserProfile = (profile, accessToken) => {
   };
 };
 
+const removeDemoUserData = async id => {
```

Comment:
why using async/await? Are they used anywhere in project? As far as I know Promises were used.